public/apps/login/login-page.tsx

```diff
@@ -212,7 +212,7 @@ export function LoginPage(props: LoginPageDeps) {
             </EuiFormRow>
           );
 
-          if (authOpts.length > 1) {
+          if (authOpts.length > 0) {
```

Comment:
Without this change, `Log in as Anonymous` button would not show with only basic auth enabled.

Reply:
extracted the condition to display anonymous login button outside this if.